web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -985,3 +985,10 @@ export type ExportedIntegrationCaResponse = {
   tls: TlsKey[];
   jwt: JwtKey[];
 };
+
+export type IntegrationDeleteWithCleanUpRequest = {
+  name: string;
+  clusterId: string;
+  subKind: 'github';
+  orgName?: string;
+};
```

Comment:
`orgName` is a GitHub-specific field.

In the future, we might have dozens of integrations, each with their own fields. If we flatten all of them here we'll have a large number of fields and it won't be clear which ones apply to which integration type.

How about something like this:

```
export type IntegrationDeleteWithCleanUpRequest = {
  name: string;
  subKind: string;
  
  gitHubOptions?: GitHubIntegrationOptions;
```